openpype/pipeline/create/context.py

```diff
@@ -1458,6 +1458,65 @@ def create(self, identifier, *args, **kwargs):
                     identifier, label, exc_info, add_traceback
                 )
             ])
+        return result
+
+    def create(
+        self,
+        creator_identifier,
+        variant=None,
+        asset_doc=None,
+        task_name=None,
+        pre_create_data=None
+    ):
+        """Trigger create of plugins with standartized
+
+        Args:
+            creator_identifier (str):
+            variant (str): Variant used for subset name.
```

Comment:
Is variant a requirement or not? :) I'm not sure. It seems optional currently due to being `variant=None`.

It'd be weird to define a subset name without a variant, no?

Reply:
It was required argument when the method was created (that's why it's first optional), then I changed it. But I don't have any valid reason why it should be optional > will change back to required.